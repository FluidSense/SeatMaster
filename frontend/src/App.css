.App {
  margin: 0;
  height: 100%;
  width: 66.68%;
  align-self: center;
}

h1{
  margin: 0;
}