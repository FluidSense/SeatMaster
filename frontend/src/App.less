.App {
  margin: 0;
  height: 100%;
 }

 h1 {
  margin: 0;
 }
