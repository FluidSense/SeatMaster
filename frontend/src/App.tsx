import * as React from 'react';
import './App.css';
import UserPage from './components/UserPage';

import { ConnectedRouter } from 'connected-react-router';
import history from './components/History';

class App extends React.Component {
  public render() {
    return (
      <ConnectedRouter history={history} >
        <UserPage />
      </ConnectedRouter>
    );
  }
}

export default App;
