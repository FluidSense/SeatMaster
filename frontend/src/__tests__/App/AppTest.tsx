import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';

import App from '../../App';

describe('<App />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<App />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
