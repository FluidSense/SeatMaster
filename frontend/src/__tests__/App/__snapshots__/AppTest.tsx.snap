// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<Connect(ConnectedRouterWithContext)
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Route
      component={[Function]}
      path="/callback"
    />
    <Connect(Presentational) />
  </Switch>
</Connect(ConnectedRouterWithContext)>
`;
